core/src/main/java/com/hcl/ecomm/core/servlets/GetCustomerCartServlet.java

```diff
@@ -37,19 +37,21 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
 			String customerToken = request.getHeader("CustomerToken");
 			String cartItems = null;
 			JSONObject responseObject = new JSONObject();
-			JSONObject responseStream = cartService.getCustomerCart(customerToken);
```

Comment:
@Subrat-Dash-416 Why this class changed in junit?

Reply:
same as above